lxd/storage/drivers/volume.go

```diff
@@ -416,7 +416,11 @@ func (v Volume) NewVMBlockFilesystemVolume() Volume {
 		newConf["size"] = v.config["size.state"]
 	} else {
 		// Fallback to the default VM filesystem size.
-		newConf["size"] = deviceConfig.DefaultVMBlockFilesystemSize
+		if v.driver.Info().Name == "powerflex" {
```

Comment:
Good suggestion. I'll add a `DefaultVMBlockFilesystemSize()` to the `Driver` interface so that the PowerFlex driver can overwrite the function to return `8GiB`.

Reply:
I think you can just add it to the struct returned from `Info()`.

Reply:
Added to the `Info` struct.